.github/workflows/CI.yml

```diff
@@ -49,25 +58,23 @@ jobs:
           Pkg.develop(url="https://github.com/JuliaDocs/DocumenterInterLinks.jl")
           Pkg.develop(path=".")
           Pkg.instantiate()
-      - name: "Run tests"
-        shell: julia --color=yes --project=test --code-coverage="@" --depwarn="yes" --check-bounds="yes" {0}
-        run: |
-          include(joinpath(pwd(), "test", "runtests.jl"))
-      - name: "Summarize coverage"
-        run: julia --project=test -L devrepl.jl -e 'display(show_coverage())'
+      - uses: julia-actions/julia-runtest@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v5
         with:
-          files: lcov.info
```

Comment:
I'm not sure that's not still needed.

Again, please keep the PR focused on dependabot